src/views/NewSettings/components/Sections/Account/Account/components/AccountDetailsModal.tsx

```diff
@@ -47,19 +47,24 @@ const AccountDetailsModal = ({
 
   const onSave = async (e: React.FormEvent<HTMLFormElement>) => {
     e.preventDefault();
+
     if (!validate(nameValue)) {
       setStatus({ tag: 'error', type: 'NAME_INVALID' });
-    } else if (!validate(lastnameValue)) {
+      return;
```

Comment:
Remember to point out to QA the parts of the code where you have made changes other than moving files around :)